authentication/urls.py

```diff
@@ -1,63 +1,80 @@
 """URL configurations for authentication"""
-from django.urls import re_path, path
+from django.conf import settings
 from django.contrib.auth import views as auth_views
+from django.urls import path, re_path
 
 from authentication.views import (
+    CustomDjoserAPIView,
+    CustomLogoutView,
     LoginEmailView,
     LoginPasswordView,
-    RegisterEmailView,
     RegisterConfirmView,
     RegisterDetailsView,
+    RegisterEmailView,
     get_social_auth_types,
-    login_complete,
-    CustomDjoserAPIView,
     get_user_details_for_keycloak,
+    login_complete,
+    post_request_password_update,
 )
-
+from open_discussions.features import KEYCLOAK_ENABLED
 
 urlpatterns = [
-    re_path(r"^api/v0/login/email/$", LoginEmailView.as_view(), name="psa-login-email"),
```

Comment:
Should we keep some or all of these around if `FEATURE_KEYCLOAK_ENABLED=False`?

```python
if not settings.FEATURE_KEYCLOAK_ENABLED:
    urlpatterns.extend([
        re_path(r"^api/v0/login/email/$", LoginEmailView.as_view(), name="psa-login-email"),
        etc
    ])
```

Similar to https://github.com/mitodl/open-discussions/pull/4234/files#diff-da53844e0b6a541573b96bec40d2b104890c238dc460ae98c829944bde64b51aR103-R120

Reply:
I have a bunch down under the `else` condition.  Do you see if I'm missing one?